app/controllers/questions/personal_info_controller.rb

```diff
@@ -5,5 +5,10 @@ def illustration_path; end
     def tracking_data
       {}
     end
+
+    def after_update_success
+      is_in_progress = current_intake.client.tax_returns.where("status >= ?", TaxReturn.statuses["intake_in_progress"]).exists?
+      current_intake.assign_vita_partner! unless is_in_progress
+    end
```

Comment:
This is a replacement for logic that I removed from `Intake#assign_partner!`

Reply:
I tried using the scope, but because it's a scope that returns a subset of clients, it doesn't work on the singular client instance returned by `current_intake.client` So I changed it to 
```ruby
    def after_update_success
      current_intake.assign_vita_partner! unless Client.after_consent.where(intake: current_intake).exists?
    end
```

Reply:
I think there are good reasons to create Client & TaxReturn records before consent, which I wouldn't want to lose track of in a future PR. Happy to talk more in-depth 1:1 if that would be interesting.